tests/publiccloud/boottime.pm

```diff
@@ -348,6 +348,9 @@ sub store_in_db {
     my ($self, $results) = @_;
     my $url = get_var('PUBLIC_CLOUD_PERF_DB_URI');
     return unless ($url);
+    my $db = get_var('PUBLIC_CLOUD_PERF_DB', 'perf');
+    my $token = get_required_var('_PUBLIC_CLOUD_PERF_DB_TOKEN');
```

Comment:
maybe we want to make this variables start with `_SECRET` ? and keep them in `workers.ini` as we doing to other public cloud creds ?

Reply:
I don't think this token is worth to keep a secret. Worst thing people can do is to push wrong metrics to our database.

Reply:
The underscore has a different purpose: When cloning the job, you don't have those variables in the clones job on your instance, so we don't spam the database with development runs.